components/Icon/Icon.stories.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from "react";
+import styled from "styled-components";
+import {
+  ChevronDownIcon,
+  SendIcon,
+  SignOutIcon,
+  ThumbsDownIcon,
+  ThumbsUpIcon
+} from "./Icon";
+
+export default {
+  title: "Icon",
+  component: SendIcon
+};
+
+export const example = () => (
+  <Catalog>
+    <ChevronDownIcon />
+
+    <SendIcon />
+
+    <SignOutIcon />
+
+    <ThumbsDownIcon />
+
+    <ThumbsUpIcon />
+  </Catalog>
+);
+
+export const withColor = () => {
+  const [isActive, setActive] = React.useState(false);
+
+  return (
+    <div>
+      <ThumbsUpIcon style={{ color: "#1dd1a1" }} />
+
+      <div
+        onClick={() => setActive(!isActive)}
+        style={{
+          display: "flex",
+          alignItems: "center",
+          color: isActive ? "#ff6b6b" : "#0f151c",
+          transition: "color 200ms ease",
+          cursor: "pointer"
+        }}
+      >
+        {"Click to change "}
```

Comment:
@axross Do you need this?
`{"Click to change "}`
Or should it be below?
`Click to change`

Reply:
That's intentional. Because the parent element is grid and not let to split text nodes